src/qililab/qprogram/qprogram.py

```diff
@@ -294,15 +304,29 @@ def play(self, bus: str, waveform: Waveform | IQPair) -> None:
         """
 
     @overload
-    def play(self, bus: str, waveform: str) -> None:
+    def play(
+        self,
+        bus: str,
+        waveform: str,
+        dwell: int | None = None,
+        delay: int | None = None,
+        repetitions: int | None = None,
+    ) -> None:
         """Play a named waveform on the bus.
 
         Args:
             bus (str): Unique identifier of the bus.
             waveform (str): An identifier of a named waveform.
         """
 
-    def play(self, bus: str, waveform: Waveform | IQPair | str) -> None:
+    def play(
+        self,
+        bus: str,
+        waveform: Waveform | IQPair | str,
+        dwell: int | None = None,
+        delay: int | None = None,
+        repetitions: int | None = None,
+    ) -> None:
```

Comment:
What are the `dwell`, `delay` and `repetitions` parameters do? Add docstrings to this method and overload methods.

Reply:
If they are only used in instrument-specific setting, like the QDAC, they should be only included in `_QdacInterface` so the users access them with `qp.qdac.play(...)`